src/servers/database.ts

```diff
@@ -550,7 +588,49 @@ export class DatabaseServer extends SkeletonServer {
       return null;
     }
   }
+  async eth_getTransactionReceiptsByBlockNumber(
+    _request: any,
+    blockNumber: web3.Quantity | web3.Tag
+  ): Promise<web3.TransactionReceipt[] | null> {
+    const blockNumber_ =
```

Comment:
Similar issue here:

Invalid block number handling in case of genesis block. Please have a look at this PR #122